elastalert/elastalert.py

```diff
@@ -199,6 +199,18 @@ def get_index(rule, starttime=None, endtime=None):
         else:
             return index
 
+    def get_six_index(self, doc_type):
```

Comment:
It's probably worth adding some comments here to explain why this is necessary in es_6. possibly with a link to https://www.elastic.co/guide/en/elasticsearch/reference/6.x/removal-of-types.html

Reply:
May also be worth adding some notes to the README about this new behavior change / migration consideration